lld/lld-c/LLDAsLibraryC.cpp

```diff
@@ -258,7 +258,7 @@ static std::string createEraVMRelLinkerScript(
 ///
 ///       /* Add padding before the metadata. Here metadata size is 32 bytes */
 ///       . = ((((. + 32) >> 5) | 1 ) << 5) - 32;
-///       *(.eravm-metadata)
+///       *(.xvm-metadata)
```

Comment:
Do we need to introduce this `xvm` here?
Can be misleading. Can we just call it `metadata` or in another LLVM-ish way?

Reply:
Sure, renamed to `metadata`.